sklearn/multiclass.py

```diff
@@ -504,7 +506,9 @@ class OneVsOneClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
         :term:`decision_function` or :term:`predict_proba`.
 
     n_jobs : int or None, optional (default=None)
-        The number of jobs to use for the computation.
+        The number of jobs to use for the computation: the n_classes
```

Comment:
In this case there are `n_classes * (n_classes - 1) / 2` OVO problems

Reply:
Hey @NicolasHug, I have modified accordingly. Can you please check the PR? Thanks!